src/index.js

```diff
@@ -49,13 +81,13 @@ class Service {
       offset: filters.$skip,
       raw: this.raw,
       distinct: true
-    }, params.sequelize);
+    }, this.computeSequelizeParam(params.sequelize, server));
 
     if (filters.$select) {
       q.attributes = filters.$select;
     }
 
-    let Model = this.applyScope(params);
+    let Model = this.applyScope(params, server);
```

Comment:
Just noticed `computeSequelizeParam()` is going to be called again from inside `applyScope()`. It might make sense to hold onto the result and pass it here. (Avoiding unexpected results if `params.sequelize` has side effects)